packages/EasyErrorHandler/src/Processors/ErrorCodesGroupProcessor.php

```diff
@@ -62,8 +65,12 @@ public function process(): ErrorCodesDto
      */
     private function determineCategoryName(array $errorCodeNames): string
     {
+        $isEnum = $this->errorCodesSource === ErrorCodesProviderLocatorInterface::SOURCE_ENUM;
```

Comment:
Until this processor knows about enums the `ErrorCodesProviderInterface` makes no sense because we still have to know how to work with different types of source. So ideally we should try to move the source-specific logic into the relevant provider and work with some high-level data only in the processor. Let's try to do it.

Reply:
Refactored. Please check it. Wdyt?

Reply:
Let's try to make it responsibility of providers.

Reply:
Refactored. Please check it